testutil/grpc.go

```diff
@@ -32,6 +32,7 @@ type RegisterFunc func(*grpc.Server)
 // establish.
 func FakeGRPCServer(tb testing.TB, registerFunc RegisterFunc) (string, *grpc.ClientConn) {
 	tb.Helper()
+	_ = 1
```

Comment:
Wonder what's the purpose of this line...

Reply:
I was testing something. I'm trying to figure out why sometimes the linter ignores files. I think it's only looking at files modified in the PR.